tools/ci-build.sh

```diff
@@ -80,7 +90,8 @@ function install_ptf_ebpf_test_deps() (
   export P4C_PTF_PACKAGES="gcc-multilib \
                            python3-six \
                            libgmp-dev \
-                           libjansson-dev"
+                           libjansson-dev\
+                           protobuf-compiler"
```

Comment:
This installation will not work. These are only for the EBPF-PTF version of the compiler, which is not enabled here.